forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -943,8 +943,8 @@ public final boolean isModal() {
         return getRules() != null && getRules().getSplitType() == CardSplitType.Modal;
     }
 
-    public final boolean hasBackSide() {
-        return isDoubleFaced() || isMeldable() || isModal();
+    public final boolean isDoubleFaced() {
+        return isTransformable() || isMeldable() || isModal();
```

Comment:
Meld technically isn't a DFC like TransformDFC or ModalDFC

Reply:
Well they should be able to be fetched though?

> There are three kinds of double-faced cards: transforming double-faced cards, modal double-faced cards, and meld cards.

Reply:
oh they are counting as DFC now, that's interesting